src/interfaces/IUniversalRouter.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.17;
+
+interface IUniversalRouter {
```

Comment:
Since this is an interface vendored from an external project, it should go in `src/lib/`

Reply:
Moved in [ced1396](https://github.com/showtime-xyz/creator-tokens/pull/44/commits/ced1396d07f469ee1a09006a368a8c1ebedb91ac)

Reply:
Cool, great! Can we also add a comment explaining where it was vendored from and ideally with a permalink to the commit it was pulled from, i.e. like [this one](https://github.com/showtime-xyz/creator-tokens/blob/main/src/lib/BondingCurveLib.sol#L7).

Reply:
I simplified the interface, but linked to the original interface here: https://github.com/showtime-xyz/creator-tokens/pull/44/commits/f7ad18cb4ad8517254750f0c8236dfae3cc72968#